flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+    "only-arches": [
+      "x86_64"
```

Comment:
why not building from source to have aarch64?

Reply:
It cannot build from source. I have tried.